doc/getting-started.rst

```diff
@@ -97,15 +98,25 @@ Install the AArch64 toolchain.
 Ubuntu 22.04
 ------------
 
-Support for Ubuntu 22.04 is still experimental.
+This is fully supported. This is recommended for the `RISC-V` host target. gcc
+version 11 is recommended for this operating system.
+
+Ubuntu 24.04
+------------
+
+This is supported, but has not been tested to the same level as 22.04. gcc
+version 13 is recommended for this operating system. This will be required for
+some qemu RISC-V features such as the half extension which was added in version
+7.
 
 Windows 10
 ----------
 
 On an x86-64 Windows 10 host system these are the minimum requirements to build
-the oneAPI Construction Kit. These tools are required for any build configuration.
+the oneAPI Construction Kit. These tools are required for any build
+configuration. Windows 11 has not been tested but may work.
```

Comment:
It seems odd to describe Windows 11 in a section on Windows 10. Ideally we would just test it though.

Reply:
Ideally we would, but it does not exist in CI (interestingly github uses windows server)

Reply:
I've gone with "should" work and put it in a different section.